examples/persons/src/Persons.re

```diff
@@ -10,29 +10,24 @@ module GetAllPersonsConfig = [%graphql
 |}
 ];
 
-module GetAllPersonsQuery = ReasonApolloHooks.Query.Make(GetAllPersonsConfig);
-
 let personsPerPage = 10;
 
 [@react.component]
 let make = () => {
   let skipRef = React.useRef(0);
-  let getAllPersons =
-    GetAllPersonsConfig.make(~skip=0, ~first=personsPerPage, ());
 
   let (_simple, full) =
-    GetAllPersonsQuery.use(
-      ~variables=getAllPersons##variables,
```

Comment:
The example fails without `variables`, since the query expects `skip` and `first` params. Was it removed by mistake? If so, I can fix it while I am at it 😄